nixos/modules/services/video/frigate.nix

```diff
@@ -195,9 +197,11 @@ in
 
     checkConfig = mkOption {
       type = bool;
-      default = pkgs.stdenv.buildPlatform.canExecute pkgs.stdenv.hostPlatform;
-      defaultText = literalExpression ''
+      default =
         pkgs.stdenv.buildPlatform.canExecute pkgs.stdenv.hostPlatform
+        && (!pkgs.stdenv.hostPlatform.isAarch64);
```

Comment:
Any reason why aarch64 is not allowed here?

Reply:
See the commit message.

Reply:
But that's a runtime issue on *some* hardware that's missing features. Don't see why it should be blocked for all aarch64 hosts.